sdcm/ec2_client.py

```diff
@@ -117,9 +237,15 @@ def terminate_instances(self, instance_ids):
                         datefmt='%H:%M:%S',
                         level=logging.INFO)
 
-    params = {'subnet_id': 'subnet-d934e980',
-              'security_group_ids': 'sg-c5e1f7a0'}
-
-    ec2 = EC2Client(params)
-    ids = ec2.create_spot_instances('m3.medium', 'ami-b390ccc8', 'us-east-1c', count=2)
-    print ids
+    network_if = [{'DeviceIndex': 0,
+                   'SubnetId': 'subnet-ec4a72c4',
+                   'AssociatePublicIpAddress': True,
+                   'Groups': 'sg-c5e1f7a0'}]
+
+    ec2 = EC2Client()
+    inst = ec2.create_spot_instances('m3.medium', 'ami-b390ccc8', 'us-east-1c', network_if, count=2)
+    print inst
+    inst = ec2.create_spot_instances('m3.medium', 'ami-b390ccc8', 'us-east-1c', network_if, count=1, duration=60)
+    print inst
+    inst = ec2.create_spot_fleet('m3.medium', 'ami-b390ccc8', 'us-east-1b', network_if, count=3)
+    print inst
```

Comment:
Maybe add a few command line options to this client, where the options default to the values you are passing here? Also, you could use logging instead of raw printing, or at least calling it the python 3 way (`print('foo')` instead of `print 'foo'`)

Reply:
The main is just for testing, this module will be used by cluster_aws.py
I can remove this main if it's confusing...